mypy.ini

```diff
@@ -7,13 +7,14 @@ ignore_missing_imports = True
 incremental = True
 strict_optional = True
 show_traceback = True
-warn_no_return = False
 warn_unused_ignores = True
 warn_redundant_casts = True
 warn_unused_configs = True
 warn_unreachable = True
 disallow_untyped_defs = true
 disallow_incomplete_defs = true
+show_error_codes = False
```

Comment:
`show_error_codes` default changed in mypy 0.990.

I'm reverting to the old behavior, because so many `.yml` test files have mypy error messages without the error code.

Reply:
Should we try to work towards `show_error_codes = True`? It feels reasonable that tests are granular enough to fail if a specific error code changes.

It could be an option to try to resolve this over time. Best thing would probably be to have a linter that disallows `type: ignore` i.e. without code in the test suite. I suppose that could be tracked in an issue.